### DISCUSSION TITLE
The website is hinderering OpenMage adoption

### DISCUSSION THREAD

**Discussion by toejam34** [OpenMage/magento-lts#4009] [General]
Folks, 

I mean the title with the greatest of respect to the developers , and users. 

I now know that if I want the very latest and greatest information that I must come to the github. However, as a new set of eyes looking inwards I will find openmage details via the webpage https://www.openmage.org/

Sadly the webpage looks out of date as it doesn't hold information like the latest releases. It would be encouraging for new people to see that we have a new release 20.8 and it was out a few days back. 

It might also be worth updating the install instructions that you can download the zip ( link to the github ) and then extract as required. 

As a final thing perhaps a extension section could be listed? Whilst I appreciate again the links are here a link from the https://www.openmage.org/ site to the freeware extensions ( or even paid ones ) might give a potential user confidence that OpenMage is the product for them. 

Its just a suggestion ; Id love this platform to thrive as I have had nothing but support from the developers when I have asked questions.

**Comment by Flyingmana** [comment]
We do appreciate this feedback, and we are also not opposed to resolve this issues.

## Regarding "the latest release", 

this could be resolved by either an automation, which updates the website on release, or by making the website on build fetching the latest release. But this will need a bit of effort.
But if anyone is able to provide PRs for this, it would be great.
If there is another idea which is easier to have the website up2date, thats also great.


## Regarding "extension section"

we already started collecting extensions in our documentation https://openmage.readthedocs.io/en/latest/modules/index.html
But the list is still a bit small, but it also gives a prepared search query for github and packagist, to easier find a more up2date list.
Also we might keep the extensions part on the website reserved for paying supporters, as the current mood under the maintainers is to rather avoid free advertising for others.

**Reply by toejam34** [reply]
Hi! 

I guess it depends on what the desire is for the website. Full automation of data pulls and posting or sometimes someone creating a new post in a wordpress powered site?  

Could a forum such as discourse replace the github + site? Therefore updates that are published by the relevant team are auto pulled into a section of discourse. Example RSS: https://github.com/OpenMage/magento-lts/releases.atom

The extensions section was a section that I wasnt aware of - perhaps as there is no route from the main site. 

What I have concern about is that the people that "know" the information are aware of its existence. Users who are somewhat new but adopters of OpenMage are not aware of these things.

**Comment by fballiano** [comment]
I've said it in the past, we don't have resources to maintain the website, so it would be better if it was a redirect to the github repo

**Comment by toejam34** [comment]
Fabrizio, Can I offer any assistance with the maintenance / running of the site? I have had so much benefit from OM I could possibly give sonething back?I have experience in Xenforo and discourse hosting / maintenance and running I think for simplicity I agree with your statement of a URL to GitHub though. Removes a layer of complexity Andy Sent from my iPhone On 27 May 2024, at 11:27, Fabrizio Balliano ***@***.***> wrote:﻿
I've said it in the past, we don't have resources to maintain the website, so it would be better if it was a redirect to the github repo

**Reply by fballiano** [reply]
thanks @toejam34, I think the website code is all in https://github.com/OpenMage/OpenMage.github.io, if you want to could create PRs to change stuff there :-)

**Comment by toejam34** [comment]
Give me some time to take a look and understand the PR system. I am a VMWare/Wintel engineer by trade so this software dev terminology is new to me. Andy Sent from my iPhone On 27 May 2024, at 11:45, Fabrizio Balliano ***@***.***> wrote:﻿
thanks @toejam34, I think the website code is all in https://github.com/OpenMage/OpenMage.github.io, if you want to could create PRs to change stuff there :-)
